lib/generator.js

```diff
@@ -20,10 +20,17 @@ const primitivesLinks = types.PRIMITIVE_TYPES.map(type => [
   `${baseUrl}${primitivesBaseUrl}${common.capitalize(type)}_type`,
 ]);
 
-const globalObjectsLinks = types.OBJECT_TYPES.map(type => [
-  type,
-  `${baseUrl}${globalObjectsBaseUrl}${type}`,
-]);
+const globalObjectsLinks = types.OBJECT_TYPES.map(type => {
+  let link = baseUrl + globalObjectsBaseUrl;
+  if (type.startsWith('Intl.')) {
+    link += type.slice(5);
+  } else if (type.startsWith('WebAssembly.')) {
+    link += `WebAssembly/${type.slice(12)}`;
```

Comment:
Why the slash here?

Reply:
Because it's part of url. Please, clarify your question.

Reply:
Oh, misunderstood the idea of the code, sorry.